### PR TITLE
fix feedback cancel button

### PR DESCRIPTION
**Pull request by AnkitPatanaik**

### CODE REVIEW

File: src/actions/feedbackActions.js [hunk 1]
```diff
@@ -33,6 +33,12 @@ export function feedbackSuccess(content, sessionId, mentorId, jwt) {
   };
 }
 
+export function cancelFeedback() {
+  return function goCancelFeedback(dispatch) {
+    return dispatch(createFeedbackSuccess());
```
Comment (jay-hankins) [inline src/actions/feedbackActions.js]: @AnkitPatanaik Why dispatch Success() in the cancel action?

Comment (AnkitPatanaik) [reply src/actions/feedbackActions.js]: i want the same functionality from the success, however i should probably create another action function to be more explicit/clear  with what's happening

Comment (jay-hankins) [reply src/actions/feedbackActions.js]: Okay, I figured it was that. I think down the road, there will be a divergence between success and failure (i.e. when we put API calls in)

**Comment by AnkitPatanaik** [comment]
@jay-hankins merge pls

**Pull request merged**